docs/src/main/tut/reference.md

```diff
@@ -82,43 +82,43 @@ Print an array of possible cell type names, as below. These names are used in ot
 |float64   |
 
 
-#### tile_dimensions
+#### rf_tile_dimensions
 
 _Python_:
 
-    Struct[Int, Int] tile_dimensions(Tile tile)
+    Struct[Int, Int] rf_tile_dimensions(Tile tile)
    
 _SQL_: `rf_tile_dimensions`
 
 Get number of columns and rows in the `tile`, as a Struct of `cols` and `rows`.
 
-#### cell_type
+#### rf_cell_type
 
 _Python_:
 
-    Struct[String] cell_type(Tile tile)
+    Struct[String] rf_cell_type(Tile tile)
     
 _SQL_: `rf_cell_type`
 
-Get the cell type of the `tile`. Available cell types can be retrieved with the @ref:[cell_types](reference.md#cell-types) function.
+Get the cell type of the `tile`. Available cell types can be retrieved with the @ref:[rf_cell_types](reference.md#rf_cell-types) function.
```

Comment:
IDK about this but i think that underscores break the links? worth checking. it's a `sed`-able change if so.

Reply:
that was a mistake... I meant to use dash.